src/main/resources/META-INF/plugin.xml

```diff
@@ -20,24 +20,35 @@
 
     <extensions defaultExtensionNs="com.intellij">
 
-        <backgroundPostStartupActivity implementation="fi.aalto.cs.intellij.activities.RequiredPluginsCheckerActivity"/>
-        <backgroundPostStartupActivity implementation="fi.aalto.cs.intellij.activities.VersionCheckerActivity"/>
-        <applicationService serviceImplementation="fi.aalto.cs.intellij.services.PluginSettings" />
+        <backgroundPostStartupActivity implementation="fi.aalto.cs.apluscourses.intellij.activities.VersionCheckerActivity"/>
+        <backgroundPostStartupActivity implementation="fi.aalto.cs.apluscourses.intellij.activities.InitializationActivity"/>
+
+        <applicationService serviceImplementation="fi.aalto.cs.apluscourses.intellij.services.PluginSettings" />
         <toolWindow id="Modules"
                     anchor="right"
-                    factoryClass="fi.aalto.cs.intellij.actions.ModulesToolWindow" />
+                    factoryClass="fi.aalto.cs.apluscourses.intellij.toolwindows.ModulesToolWindowFactory" />
         <toolWindow id="Exercises"
                     anchor="right"
-                    factoryClass="fi.aalto.cs.intellij.actions.ExercisesToolWindow" />
+                    factoryClass="fi.aalto.cs.apluscourses.intellij.toolwindows.ExercisesToolWindowFactory" />
     </extensions>
 
     <actions>
         <group id="A+.Menu" text="A+" description="A+ Menu">
```

Comment:
Nit: it would probably make sense to unify the approach to assigning IDs to `groups` (and `actions`)